dandischema/tests/test_datacite.py

```diff
@@ -71,7 +71,7 @@ def _basic_publishmeta(dandi_id, version="v.0", prefix="10.80507"):
             "schemaKey": "PublishActivity",
         },
         "version": version,
-        "doi": f"{prefix}/dandi.{dandi_id}.{version}",
+        "doi": f"{prefix}/dandi.{dandi_id}/{version}",
```

Comment:
i think `/` is cleaner and still allowed, since our tests pass with datacite, but no strong opinion on my side other than it's a nice semantic break.

we just have to decide and then make sure @dchiquito uses that.